Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node
+ARG SERVICE_NAME
+
+# These steps ensure that npm install is only run when package.json changes
+RUN mkdir -p /usr/src/$SERVICE_NAME
+ADD ./package.json /usr/src/$SERVICE_NAME/package.json
+WORKDIR /usr/src/$SERVICE_NAME
+RUN npm install --production
+
+ADD . /usr/src/$SERVICE_NAME
```

Comment:
If we're including Dockerfiles with this service, should we just write the service name in instead of taking in a command line argument? This would apply to users service as well.

What do you think?